emails/views.py

```diff
@@ -738,6 +746,67 @@ def _strip_localpart_tag(address):
     return f"{subaddress_parts[0]}@{domain}"
 
 
+def _convert_html_content(
+    html_content: str,
+    to_address: str,
+    from_address: str,
+    language: str,
+    has_premium: bool,
+    sample_trackers: bool,
+    remove_level_one_trackers: bool,
+    now: datetime | None = None,
+) -> tuple[str, int]:
+    # frontend expects a timestamp in milliseconds
+    now = now or datetime.now(timezone.utc)
+    datetime_now_ms = int(now.timestamp() * 1000)
+    # user_profile = address.user.profile
```

Comment:
change (blocking): We try to remove commented lines, right?

Reply:
yep, supposed to be removed...